service/src/main/java/bio/terra/workspace/service/resource/controlled/model/ControlledResource.java

```diff
@@ -283,7 +283,8 @@ public ControlledResourceFields buildControlledCloneResourceCommonFields(
       @Nullable String name,
       @Nullable String description,
       String createByEmail,
-      String region) {
+      String region,
+      @Nullable String applicationId) {
```

Comment:
Why allow this to be null?

Reply:
Same as above; controlled resources _may_ be managed by an app (but certainly not always). For application managed resources that are being cloned, we need to ensure the application ID propagates.